dashboard/modules/metrics/metrics_head.py

```diff
@@ -66,12 +106,25 @@ def _create_default_grafana_configs(self):
         ) as f:
             f.write(GRAFANA_DATASOURCE_TEMPLATE.format(prometheus_host=prometheus_host))
 
+        # Output the dashboards in a special directory
+        if self._grafana_dashboard_output_dir:
```

Comment:
this code block is to support the custom config?

Reply:
no, it's so users can customize the directory that the dashboards output to. Necessary for the product to work e2e.